
Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили
результаты по метрике конверсий в покупку.
Выберите метод оценки и оцените есть ли статистически значимые различия между
конверсиями в двух группах. Дайте краткие рекомендации команде.

Результаты:

1) Число юзеров в группах, которые заходили на сайт в период эксперимента:
n1 = 15 550 и n2 = 15 550
2) Число юзеров в группах, которые совершили хотя бы одну покупку
за период эксперимента: n1 = 164 и n2 = 228
3) Конверсии: conv1 = 1.05% conv2 = 1.47% .

При α=5% можно использовать Z-критерий для разности долей. Формула Z-критерия:

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))

где p = (n1 * p1 + n2 * p2) / (n1 + n2)

p1 и p2 - конверсии в группах, N1 и N2 - количество юзеров в группах, которые заходили на сайт в период эксперимента, n1 и n2 - количество юзеров, которые совершили хотя бы одну покупку за период эксперимента.

In [ ]:
from math import sqrt

In [ ]:
p1 = 164 / 15550

p2 = 228 / 15550

n1 = 164

n2 = 228

N1 = 15550

N2 = N1

In [ ]:
p =  (n1 * p1 + n2 * p2) / (n1 + n2)

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))
Z

-3.2110930329358367

Так как расчетное значение Z меньше критического значения, т.е. -3.2110930329358367 < 1.96 , то мы делаем вывод, что статистически значимые различия в группах существуют.
На основании этих данных, мы делаем выводы: упрощение формы приводит к увеличению конверсии в покупку.
Рекомендация: применить использование упрощеннной формы заказа.